packages/@react-spectrum/listbox/docs/ListBox.mdx

```diff
@@ -423,7 +423,7 @@ import {theme} from '@react-spectrum/theme-default';
 import {User} from '@react-spectrum/test-utils';
 
 let testUtilUser = new User({interactionType: 'mouse'});
-// ...
+// Other setup, be sure to check out the suggested mocks mentioned above in https://react-spectrum.adobe.com/react-spectrum/ListBox.html#testing
```

Comment:
Hopefully makes it a little more clear that additional setup might be needed for RSP components in general